### PR TITLE
Everforest 🌲 for RStudio

### PR DESCRIPTION
**Pull request by eric-hunt**
@gadenbuie,
Last one.. for a while.. probably. 😄 Dark and light themes in hard, medium, and soft variants.
dark_medium
light_medium

### CODE REVIEW

File: inst/templates/everforest.R [hunk 1]
```diff
@@ -0,0 +1,578 @@
...
+#' (hard, medium, soft); see [Everforest](https://github.com/sainnhe/everforest)
+#' @param apply A Boolean - generate and install/apply the .rstheme file (TRUE),
+#' or just save the .rstheme file without applying (FALSE); defaults to FALSE
+#' @param as_sass A Boolean - save theme as .sass file and do not apply (TRUE),
+#' defaults to FALSE
+#' @param ... Additional arguments to pass to [rsthemes::rstheme()] overriding
+#' defaults if necessary
+#'
+#' @export
+everforest_rstheme <- function(
```
Comment (gadenbuie) [inline inst/templates/everforest.R:156]: Can you please also apply the feedback from #102 here?

File: inst/templates/everforest.R [hunk 2]
```diff
@@ -0,0 +1,578 @@
...
+    "
+    .rstudio-themes-flat .gwt-TabLayoutPanelTab-selected {
+      .gwt-TabLayoutPanelTabInner .rstheme_tabLayoutCenter {
+        box-shadow: 0 3px 0 $line_1 inset;
+        border-radius: 0 !important;
+
+        .gwt-Label {
+          \\ font-weight: 600;
+          \\ color: $fg;
+        }
```
Comment (gadenbuie) [inline inst/templates/everforest.R:473]: Same feedback from #102 (just remove?)

```suggestion
```

File: inst/templates/everforest.R [hunk 3]
```diff
@@ -0,0 +1,578 @@
...
+  c(TRUE, FALSE),
+  function(dark) {
+    purrr::walk(
+      c("hard", "medium", "soft"),
+      function(variant) {
+        everforest_rstheme(dark = dark, variant = variant)
+      }
+    )
+  }
+)
```
Comment (gadenbuie) [inline inst/templates/everforest.R:578]: Also from #102, we can remove this and rely on `inst/template/make.R`.

**Review by gadenbuie — COMMENTED** [review]
Very cool, thank you! Most of the feedback in #102 applies here too 😄

**Comment by eric-hunt** [comment]
Applied the same changes as in #102. Should be all set now. Thanks!

**Review by gadenbuie — APPROVED** [review]
Thanks again @eric-hunt!

**Pull request merged**